infra/pr_helper.py

```diff
@@ -35,14 +35,21 @@
 
 def get_criticality_score(repo_url):
   """Gets the criticality score of the project."""
+  if repo_url.endswith('.git'):
+    repo_url = repo_url[:-4]
   report = subprocess.run([
       CRITICALITY_SCORE_PATH, '--format', 'json',
       '-gcp-project-id=clusterfuzz-external', '-depsdev-disable', repo_url
   ],
                           capture_output=True,
                           text=True)
 
-  report_dict = json.loads(report.stdout)
+  try:
+    report_dict = json.loads(report.stdout)
+  except:
+    print(f'stdout: {report.stdout}')
```

Comment:
Can we add "criticality score" to these strings to make it clearer where these stdout/stderr are coming from?